frontend/components/footer.tsx

```diff
@@ -56,7 +51,6 @@ export default function Footer({ baseUrl }) {
         />{' '}
         {}
         <object
-          alt="chat on Discord"
```

Comment:
What's the thinking behind removing these?

Reply:
It throws an error in TypeScript. I think the error is correct, since [`<object>` doesn't seem to support `alt` attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/object).